package.json

```diff
@@ -19,7 +19,7 @@
     "serverless-wsgi": "^1.3.0"
   },
   "dependencies": {
-    "npm": "^5.7.1",
+    "npm": "^5.10.0",
```

Comment:
Do we need `npm` in your dependencies here, or should this be using the [`engines`](https://docs.npmjs.com/files/package.json#engines) field?